functions/src/modules/pm/categories/prizes/listPrize.ts

```diff
@@ -0,0 +1,57 @@
+import type { ExpressFunction } from "../../../../@types";
+import { firebaseApp } from "../../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../../helpers";
+import { uasPermissionSwitch } from "../../../../systems/uas";
+
+const listPrize: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: {
+      accepted: execute,
+     
+    },
+  })(req, res, next);
+};
+
+const execute: ExpressFunction = (req, res, next) => {
+  const errorHandler = expressErrorHandlerFactory(req, res, next);
+
+  firebaseApp
+    .firestore()
+    .collection("Prize")
+    .orderBy("prizeId", "asc")
+    .get()
+    .then((documents) => {
+      send(documents.docs)(req, res, next);
+    })
+    .catch(errorHandler);
+};
+
+
+
+// const executeIfApproved: ExpressFunction = (req, res, next) => {
+//   const errorHandler = expressErrorHandlerFactory(req, res, next);
+
+//   firebaseApp
+//     .firestore()
+//     .collection("Prize")
+//     .orderBy("prizeName", "asc")
+//     .where("approvalStatus", "==", "approved")
+//     .get()
+//     .then((documents) => {
+//       send(documents.docs)(req, res, next);
+//     })
+//     .catch(errorHandler);
+// };
+
+const send: (
+  docs: FirebaseFirestore.QueryDocumentSnapshot<FirebaseFirestore.DocumentData>[]
+) => ExpressFunction = (docs) => (req, res, next) => {
+  const Prize: PMPrize[] = [];
```

Comment:
This should be lower case and plural.
```suggestion
  const prizes: PMPrize[] = [];
```